modules/gapi/src/backends/ie/util.hpp

```diff
@@ -27,5 +30,6 @@ GAPI_EXPORTS InferenceEngine::Blob::Ptr to_ie(cv::Mat &blob);
 
```

Comment:
Since this file is internal now, please put a note here why these functions are `EXPORT`ed (to make them accessible by the test suite)